### ISSUE TITLE
Don't import ggplot2?

### ISSUE THREAD

**Issue by hadley** [sjmgarnier/viridis#40]
Because then I could use it from ggplot2 to replace the current continuous colour scheme

https://github.com/tidyverse/ggplot2/issues/1823

**Comment by sjmgarnier** [comment]
@hadley Would [viridisLite](https://github.com/sjmgarnier/viridisLite) be what you're looking for?

**Comment by hadley** [comment]
Yes, although it concerns me that viridis does not appear to import viridisLite.

**Comment by sjmgarnier** [comment]
@hadley I guess this is an historical oversight. We wrote and released viridis first, with the ggplot2 scales included (hence the ggplot2 dependency). Then someone asked us to make a more basic version because they didn't want their packages to depend on ggplot2. So we just stripped down viridis to make viridisLite. I guess we could have been smarter and do as you suggest. I will try to fix this when I find the time.
